libraries/core-react/src/Popover/PopoverItem.tsx

```diff
@@ -0,0 +1,199 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React, { useEffect, useRef, forwardRef, HTMLAttributes } from 'react'
+import styled, { css } from 'styled-components'
+import { Icon } from '../Icon'
+import { Card } from '../Card'
+import { Button } from '../Button'
+import { close } from '@equinor/eds-icons'
+import { spacingsTemplate, typographyTemplate } from '../_common/templates'
+import { useCombinedRefs } from '../_common'
+
+import { popover as tokens, Placement } from './Popover.tokens'
+
+Icon.add({ close })
```

Comment:
Suggest dropping the `Icon.add` in favour of just setting the `data` property on `<Icon/>` as its a cleaner approach now.

Example;` <Icon data={close} title="close" size={48} />`

Reply:
👍